docs/en/04_Changelogs/4.0.0.md

```diff
@@ -92,6 +93,138 @@ For a full list of renamed classes, check the `.upgrade.yml` definitions in each
 
 The rename won't affect class-based permission codes or database table names.
 
+##### Use ::class whenever possible
+
+Prefer:
+ ```php
```

Comment:
I take your point on this particular example, but there's lot of others further down with a clearer before/after relationship. Also, "prefer/instead" feels like the wrong way around for code examples. If you compare it to `.env`, we've first documented the old way, then the new way.